docs/deployment/index.md

```diff
@@ -17,14 +17,23 @@ _For more information about advanced configuration, read our documentation [here
 ```
 
 # Kubernetes
-We currently provide [a sample declaration](https://github.com/tomkerkhove/promitor/tree/master/deploy) which deploys all the required infrastructure on your Kubernetes cluster.
+We currently provide [a helm chart](https://github.com/tomkerkhove/promitor/tree/master/charts) which deploys all the required infrastructure on your Kubernetes cluster.
```

Comment:
Cool, but how do I deploy this chart? I'm new to Helm and I'm probably not the only one.

Can we add a doc on how to do this?

Reply:
Should be clearer w/ new commit

Reply:
👌